packages/manager/apps/public-cloud/src/sidebar/sidebar.less

```diff
@@ -0,0 +1,143 @@
+@background-color: rgb(244,248,251);
+@text-color: rgb(17,63,109);
+
+cloud-sidebar { /* stylelint-disable-line */
+  background-color: @background-color;
+  color: @text-color;
+  width: 100%;
+  height: 100%;
+
+  oui-dropdown-content { /* stylelint-disable-line */
+    color: @text-color;
+    width: 90%;
+
+    ul {
+      list-style-type: none;
+      padding: 0;
+      margin: 0;
+      max-height: 50vh;
+      overflow: auto;
+
+      li {
+        border-bottom: 1px solid rgb(230, 245, 252);
+        padding: 0 0 4pt;
```

Comment:
Could you explain why you are using `pt` unit values when dealing with margin/padding?

Because sometimes there are `rem` units which is also used to set margin/padding.

Reply:
-> We should use only rem, not pt but design tools give us only pt.

Should be converted to rem ;)